auth-api/src/auth_api/resources/affiliation_invitation.py

```diff
@@ -0,0 +1,167 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""API endpoints for managing an Affiliation Invitation resource."""
+
+from flask import request
+from flask_restx import Namespace, Resource, cors
+
+from auth_api import status as http_status
+from auth_api.auth import jwt as _jwt
+from auth_api.exceptions import BusinessException
+from auth_api.schemas import utils as schema_utils
+from auth_api.services import AffiliationInvitation as AffiliationInvitationService
+from auth_api.services import User as UserService
+from auth_api.tracer import Tracer
+from auth_api.utils.roles import Role
+from auth_api.utils.util import cors_preflight
+
+
+API = Namespace('invitations', description='Endpoints for affiliation invitations management')
+TRACER = Tracer.get_instance()
+
+
+@cors_preflight('GET,POST,OPTIONS')
+@API.route('', methods=['GET', 'POST', 'OPTIONS'])
+class AffiliationInvitations(Resource):
+    """Resource for managing AffiliationInvitations."""
+
+    @staticmethod
+    @TRACER.trace()
+    @cors.crossdomain(origin='*')
+    @_jwt.has_one_of_roles(
+        [Role.SYSTEM.value, Role.STAFF_VIEW_ACCOUNTS.value, Role.PUBLIC_USER.value])
+    def get():
+        """Get affiliation invitations."""
+        try:
+            org_id = request.args.get('orgId', None)
+            status = request.args.get('status', None)
+            data = AffiliationInvitationService.search_invitations_for_from_org(org_id, status)
+
+            if data is None:
```

Comment:
`data = data or []` maybe?

Reply:
`search_invitations_for_from_org` function returns a dictionary with a key 'affiliationInvitations' that maps to a list.
maybe `data = data or {'affiliationInvitations': []}` ?

Reply:
Good catches thanks! Updated as per Jia's comment